docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -326,6 +326,35 @@ this table:
 
 No other types are supported.
 
+## Delta Lake table features
+
+The connector supports the following [Delta Lake table 
+features](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#table-features):
+
+:::{list-table} Table features
+:widths: 70, 30
+:header-rows: 1
+
+* - Feature
+  - Description
+* - Append-only tables
+  - Writers only
```

Comment:
I'd consider rephrasing this as "Fully supported" since this feature only needs writer support.